src/main/java/org/mycore/externalstore/model/MCRExternalStoreRawSettingsWrapper.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.externalstore.model;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Wrapper model for remote store config/settings.
+ */
+public class MCRExternalStoreRawSettingsWrapper {
+
+    private String content;
+
+    private boolean encryptedContent;
+
+    /**
+     * Constructs new settings wrapper.
+     * Necessary for {@link ObjectMapper}.
+     */
+    @SuppressWarnings("PMD.UnnecessaryConstructor")
```

Comment:
Can be removed if there is no other constructor and it will be generated by the compiler